docs/user_guide/assessment.rst

```diff
@@ -216,6 +216,68 @@ case we see that the subgroup ``(a, 8)`` has a result of ``NaN``, indicating
 that there were no samples in it.
 
 
+Scalar Results from :code:`MetricFrame`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Higher level machine learning algorithms (such as hyperparameter tuners) often
+make use of metric functions to guide their optimisations.
+Such algorithms generally work with scalar results, so if we want the tuning
+to be done on the basis of our fairness metrics, we need to perform aggregations
+over the :class:`.MetricFrame`.
+
+We provide a convenience function, :func:`fairlearn.metrics.make_derived_metric`
+to generate scalar-producing metric functions based on the aggregation methods
+mentioned above (:meth:`.MetricFrame.group_min`, :meth:`.MetricFrame.group_max`,
+:meth:`.MetricFrame.difference`, and :meth:`.MetricFrame.ratio`).
+This takes an underlying metric function, the name of the desired transformation, and
+optionally a list of parameter names which should be treated as sample parameters.
+The result is a function internally builds the :class:`.MetricFrame` and performs
+the requested aggregation. For example:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import make_derived_metric
+    >>> fbeta_difference = make_derived_metric(metric=skm.fbeta_score,
+    ...                                        transform='difference')
+    >>> fbeta_difference(Y_true, Y_pred, beta=0.7,
+    ...                  sensitive_features=group_membership_data)
+    0.7525252525...
```

Comment:
Fair point - that was a bit excessive on the significant figures.

For '32bit ones' should I be reading "x87 FPU" or did something else have its own approach to floating point?

Reply:
more like 32bit windows machines which usually give us a headache, or 32bit ARM CPUs out there, of which there are still plenty

Reply:
Well, I suppose that recent `numpy` issue is a counter example, but handling of double precision _shouldn't_ be affected by whether the underlying CPU is 32 or 64 bit. The case I know about is the x87 FPU, which was 80 bit internally, but SSE has been around since the Pentium Pro (give or take.... it's been a while).

Reply:
Well, I know it _shouldn't_, but in my experience for some reason (probably CPU dependent data types in the underlying BLAS code) it affects the results every now and then.

Reply:
Theory must certainly yield to reality :-)